tests/10-create.js

```diff
@@ -0,0 +1,286 @@
+  this.implemented = [...match.keys()];
+  this.notImplemented = [...nonMatch.keys()];
+  this.rowLabel = 'Test Name';
+  this.columnLabel = 'Did Key Resolver';
+  // the reportData will be displayed under the test title
+  this.reportData = reportData;
+  for(const [columnId, implementation] of match) {
+    const didResolver = implementation.didResolvers.find(
+      dr => dr.tags.has('Did-Key'));
+    const makeUrl = did =>
+      `${didResolver.settings.endpoint}/${encodeURIComponent(did)}`;
+    describe(columnId, function() {
+      it('The scheme MUST be the value `did`', async function() {
+        this.test.cell = {columnId, rowId: this.test.title};
+        const {scheme} = splitDid({did});
+        should.exist(scheme, 'Expected first part of the did to exist.');
+        scheme.should.be.a('string', 'Expected did scheme to be a string.');
+        scheme.should.equal('did', 'Expected scheme to be "did"');
+      });
+      it('MUST raise `invalidDid` error if scheme is not `did`',
+        async function() {
+          this.test.cell = {columnId, rowId: this.test.title};
+          const noDidScheme = did.replace(/^did:/, 'notDid');
+          const {result, error} = await didResolver.get({
+            url: makeUrl(noDidScheme),
+            headers
+          });
+          shouldErrorWithData(result, error);
+          const {data} = error;
+          shouldBeDidResolverResponse(data);
+          shouldHaveDidResolutionError(data, 'invalidDid');
+        });
+      it('The method MUST be the value `key`', async function() {
+        this.test.cell = {columnId, rowId: this.test.title};
+        const {method} = splitDid({did});
+        should.exist(method, 'Expected did to have a method');
+        method.should.be.a('string', 'Expected method to be a string');
+        method.should.equal('key', 'Expected method to equal key');
+      });
+      it('MUST raise `invalidDid` error if method is not `key`',
+        async function() {
+          this.test.cell = {columnId, rowId: this.test.title};
+          const {parts} = splitDid({did});
+          // use the first part and everything after the second part
+          const methodNotKey = `${parts[0]}:notKey:${parts.slice(2).join(':')}`;
+          const {result, error} = await didResolver.get({
+            url: makeUrl(methodNotKey),
+            headers
+          });
+          shouldErrorWithData(result, error);
+          const {data} = error;
+          shouldBeDidResolverResponse(data);
+          shouldHaveDidResolutionError(data, 'invalidDid');
+        });
+      it('The version MUST be convertible to a positive integer value.',
+        async function() {
+          this.test.cell = {columnId, rowId: this.test.title};
+          const {version} = splitDid({did});
+          should.exist(version, `Expected ${did} to have a version.`);
+          shouldHaveValidVersion(version);
+        });
+      it('MUST raise `invalidId` if version is not convertible to a ' +
```

Comment:
Can we change `invalidId` to `invalidDid`?

Reply:
you are actually correct. The correction was made here: https://github.com/w3c-ccg/did-key-test-suite/pull/19/commits/e8f1ee9d76a5b999663c4ec3406feea41d04c27a

This came from the original PR: https://github.com/w3c-ccg/did-key-test-suite/pull/17/files
Which has `INVALID_ID` which doesn't match the error in the spec:
https://w3c-ccg.github.io/did-method-key/

so THANK YOU  great catch!